{% extends "layout.html" %}
{% block content %}
<div class="container">
  <h1>
    <strong>
      Documentation
    </strong>
  </h1>
  <div class="row">
    <div class="col-md-12">
      <h3>Abstract</h3>
      <p>
          Mechanistic molecular studies in biomedical research often discover important genes that are aberrantly
          expressed in disease. However, manipulating these genes’ expression to attempt to improve the disease state is
          challenging. Here we present Dr. Gene Budger (DGB), a web-based application developed to assist investigators
          with prioritizing small molecules that are predicted to maximally influence the expression of their gene of
          interest. With DGB users can enter a gene symbol, and whether they wish to upregulate or downregulate its
          expression. The output of the application is a ranked list of small molecules that have been experimentally
          determined to produce the desired expression effect. The table includes log-transformed fold changes and
          p-values for each small molecule, reporting the significance of differential expression as determined by the
          limma method [1]. Relevant links are provided to further explore knowledge about the target gene, the small
          molecule, and the source of evidence from which the relationship between the small molecule and the target gene
          as derived. The experimental data contained within DGB is compiled from signatures extracted from the LINCS
          L1000 dataset and the Gene Expression Omnibus (GEO) [2, 3]. DGB provides a useful preliminary technique for
          identifying small molecules that can target the expression of  single genes in mammalian cells.
      </p>

      <br>

      <h3>References</h3>
      <p>
          1. Ritchie, M. E., B. Phipson, D. Wu, Y. Hu, C. W. Law, W. Shi, and G. K. Smyth. “limma powers differential expression analyses for RNA-sequencing and microarray studies”. Nucleic Acids Research 43, no. 7 (April 2015): e47-e47<br>
          2. Wang, Z. et al. “Extraction and Analysis of Mammalian Gene Expression Signatures from GEO by the Crowd”. Nature Communications 7 (2016): 12846<br>
          3. Duan, Q. et al. “L1000CDS2: LINCS L1000 characteristic direction signatures search engine.” npj Systems Biology and Applications. 2, 16015 (2016)<br>
      </p>

      <br>

      <h3>Terms of Use</h3>
      <p>
        DGB does not collect and use any user data.
        <br />
        <br />
        DGB is provided AS-IS and should be used for research purpose only. Members of the Ma’ayan Lab are not reponsible for the accuracy, applicability, reliability and completion of any of the predictions made by DGB. Members of the Ma’ayan Lab are not responsible for any conclusions drawn from the results pages by users.
        <br />
        <br />
        Free for academic, non-profit use, but for commercial uses please contact Mount Sinai Innovation Partners at <a href="http://www.ip.mountsinai.org/" target="_blank">http://www.ip.mountsinai.org/</a> for a license.
      </p>
    </div>
  </div>
</div>
{% endblock %}
