{% extends "layout.html" %}
{% block content %}
<div class="container">
  <h1>
    <strong>
      Documentation
    </strong>
  </h1>
  <div class="row">
    <div class="col-md-12">
      <h3>Abstract</h3>
      <p>
          Mechanistic molecular studies in biomedical research often discover important genes that are aberrantly expressed in disease. However, manipulating these genes in an attempt to improve the disease state is challenging. Here we reveal Drug Gene Budger (DGB), a web-based and mobile application developed to assist investigators in order to prioritize small molecules that are predicted to maximally influence the expression of their target gene of interest. With DGB, users can enter a gene symbol along with the wish to upregulate or downregulate its expression. The output of the application is a ranked list of small molecules that have been experimentally determined to produce the desired expression effect. The table includes log-transformed fold change, p-value and q-value for each small molecule, reporting the significance of differential expression as determined by the limma method. Relevant links are provided to further explore knowledge about the target gene, the small molecule, and the source of evidence from which the relationship between the small molecule and the target gene was derived. The experimental data contained within DGB is compiled from signatures extracted from the LINCS L1000 dataset, the original Connectivity Map (CMap) dataset, and the Gene Expression Omnibus (GEO). DGB also presents a specificity measure for a drug-gene connection based on the number of genes a drug modulates. DGB provides a useful preliminary technique for identifying small molecules that can target the expression of a single gene in human cells and tissues.
      </p>

      <br>

      <h3>Citation</h3>
      <p>
        Wang, Z. He E., Sani K., Jagodonik K. M., Silverstein M., Ma'ayan A. <a target="_blank" href="https://doi.org/10.1093/bioinformatics/bty763">"Drug Gene Budger (DGB): An application for ranking drugs to modulate a specific gene based on transcriptomic signatures"</a> <i>Bioinformatics</i>, bty763 (2018)
      </p>

      <h3>References</h3>
      <p>
          1. Ritchie, M. E., B. Phipson, D. Wu, Y. Hu, C. W. Law, W. Shi, and G. K. Smyth. “limma powers differential expression analyses for RNA-sequencing and microarray studies”. <i>Nucleic Acids Research</i> 43, no. 7 (April 2015): e47-e47<br>
          2. Wang, Z. et al. “Extraction and Analysis of Mammalian Gene Expression Signatures from GEO by the Crowd”. <i>Nature Communications</i> 7 (2016): 12846<br>
          3. Duan, Q. et al. “L1000CDS2: LINCS L1000 characteristic direction signatures search engine.” <i>npj Systems Biology and Applications</i>. 2, 16015 (2016)<br>
      </p>

      <br>

      <h3>Terms of Use</h3>
      <p>
        DGB does not collect and use any user data.
        <br />
        <br />
        DGB is provided AS-IS and should be used for research purpose only. Members of the Ma’ayan Lab are not reponsible for the accuracy, applicability, reliability and completion of any of the predictions made by DGB. Members of the Ma’ayan Lab are not responsible for any conclusions drawn from the results pages by users.
        <br />
        <br />
        Free for academic, non-profit use, but for commercial uses please contact Mount Sinai Innovation Partners at <a href="http://www.ip.mountsinai.org/" target="_blank">http://www.ip.mountsinai.org/</a> for a license.
      </p>
    </div>
  </div>
</div>
{% endblock %}
